R/fload.R

```diff
@@ -196,7 +196,7 @@ fload <- function(json,
     )
 
     if (always_list && length(json) == 1L) {
-        `attr<-`(list(out), "names", names(json))
+        `names<-`(list(out), names(json))
```

Comment:
`setNames()` is the more idiomatic way to set names for an object.

Reply:
I suppose, but it's the exact same behavior without needing to add `{stats}` to the `DESCRIPTION`'s `Imports:`.

Reply:
Maybe, maybe not. `setNames()` is from `stats`, reading help suggests it is "just a short form" for assigning and naming.  I am cool with `base:names()`.  Can't argue with `base` even if there are shortcuts :)

Reply:
Indeed, I forgot about the fact that `setNames` comes from **stats**. Sorry for the noise :)